src/about_alan.md

```diff
@@ -1,20 +1,19 @@
 # About `alan`
 
-`alan` is meant to fit like a glove. Like a glove, it's meant to be familiar and obvious how to use and generally not restricting. But like a glove, it has a few restrictions at the edges that going without it wouldn't have. And also like a glove, `alan` lets you grasp at things that would otherwise hurt you if you tried. ;)
+`alan` is a natively-parallel, statically-compiled, type-inferred language with a familiar syntax and many compile-time and run-time safety guarantees.
 
-## `alan`'s not turing
+* The compiler can determine which array operations are safely parallelizable and automatically distributes them across a threadpool.
+* The runtime can determine which operations are IO operations and automatically `async/await` them, as well as automatically batch IO operations that can be executed in parallel safely.
+* The type system enforces safe code to prevent most runtime errors (out-of-memory being a notable exception).
+* The module system has a built-in mocking mechanism (with no runtime performance penalty) that can be used for fine-grained permissioning of access to the standard library for third-party libraries.
+* Aggressive function inlining and dead-code removal to make sure unused code isn't even available in the output to potentially exploit.
 
-The `alan` programming language is a little bit different from most programming languages, in that it is intentionally not Turing-complete. Loops and recursion have been eliminated[^1] from the language. But this doesn't make it a dumb declarative thing like HTML or eBPF[^2]. Instead, substitutes for looping and recursion have been provided by the language that should cover all[^3] of your needs in a controlled way, such that it should be impossible[^4] to write code that doesn't halt.
+To accomplish this, `alan` makes one single, significant trade-off versus other programming languages: Functions in `alan` are all acyclic directed graphs. No arbitrary loops or recursion are allowed.
```

Comment:
technically handlers, but I think that is fine unless we wanted to call out the event loop here

Reply:
I think this sort of introduction to `alan` would be ill-served by introducing potentially-confusing terminology that has a very particular meaning within `alan` but not elsewhere.